tests/firedrake/regression/test_nullspace.py

```diff
@@ -323,6 +324,10 @@ def test_near_nullspace_mixed(aux_pc):
 
     f = as_vector((0, -9.8*conditional(inside_box, 2, 1)))
     L = inner(f, v)*dx
+    if rhs == 'cofunc_rhs':
+        L = assemble(L)
```

Comment:
Is this test passing without bcs here?

Reply:
The test passed. Were you not expecting it to?